### DISCUSSION TITLE
gh actions-importer keeps calling github.com instead of GHES instance

### DISCUSSION THREAD

**Discussion by tyrken** [github/gh-actions-importer#294] [Feedback] [answered]
We're trying to look at using the importer to convert some CircleCI config for repos on our GH Enterprise server, but despite using "https://git.mycorp.com/" as the `GITHUB_INSTANCE_URL` when running `gh actions-importer configure` or the `--github-instance-url` param with `gh actions-importer audit circle-ci`, it initially errored with:

```
Message: Unable to locate credentials for 'https://github.com'
```

When I provide said alternative credentials via a `--credentials-file` it uses them to access our github.com org & enumerate public & private repos there where we have a few - but they are old, not under CI control & not the ones we want!

Has anybody managed to use this with GHES?

**Comment by mh-isaac-riceweber** [comment, marked as answer]
Looks like the [documentation](https://docs.github.com/en/enterprise-server@3.12/actions/migrating-to-github-actions/automated-migrations/migrating-from-circleci-with-github-actions-importer#using-environment-variables) for the environment variables is incorrect in the use of fallbacks.  According to the doc, when you don't have `CIRCLE_CI_SOURCE_GITHUB_INSTANCE_URL` defined, it should fall back to use `GITHUB_INSTANCE_URL`.  I found this to not be the case (consistent with your experience).

I was able to get an audit for an org in our self-hosted GHES instance by defining `CIRCLE_CI_SOURCE_GITHUB_INSTANCE_URL` in my `.env.local` file.  To be safe, I also defined `CIRCLE_CI_SOURCE_GITHUB_ACCESS_TOKEN` to have the same value as `GITHUB_ACCESS_TOKEN`.

**Reply by mh-isaac-riceweber** [reply]
As an aside, I didn't actually find this from the documentation site. Rather I dug into the ruby code contained in the docker image to figure out where the github base url was coming from.

**Reply by tyrken** [reply]
Thanks @mh-isaac-riceweber - that's working correctly for me now (with just `GITHUB_ACCESS_TOKEN`, no need for any `CIRCLE_CI_SOURCE_GITHUB_ACCESS_TOKEN`)!
